integrations/apollo/src/index.ts

```diff
@@ -187,7 +187,10 @@ export const createApolloClient = ({
 
   interpreter?.onTransition(async (state, event) => {
     if (['SIGNOUT', 'SIGNED_IN', 'TOKEN_CHANGED'].includes(event.type)) {
-      if (event.type === 'SIGNOUT') {
+      if (
+        event.type === 'SIGNOUT' ||
+        (event.type === 'TOKEN_CHANGED' && state.context.accessToken === null)
```

Comment:
I think this fix isn't working. When I debug this I see that `accessToken` is an object. So shouldn't we check the `value` here? Like this:

```ts
(event.type === 'TOKEN_CHANGED' && state.context.accessToken.value === null)
```